src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml

```diff
@@ -7,6 +7,16 @@
 	mc:Ignorable="d"
 	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
+	<Page.Resources>
```

Comment:
Why did you change that file ? Is it for another PR ?

Reply:
I added two Ellipses in this file, one with a Fill that uses a SolidColorBrush with opacity and the other a Stroke that uses one. To verify that shapes in WebAssembly function with the changes as well.